samples/highcharts/demo/combo-data-resolution/demo.details

```diff
@@ -0,0 +1,14 @@
+---
+name: 'Same data, different intervals'
```

Comment:
Technically not the same data (though it's aggregated from the same source). A more descriptive title would be "Same x-axis, different intervals"